### PR TITLE
copy edits to the C++ examples on the website

### LINKED ISSUE #948: Copy edits to the CUDA Quantum C++ Examples page and code
typos and style issues are itemized in this [google document](https://docs.google.com/document/d/1_9itCtIFOhik7HX8VSNVYrLJr9Y1XErRhmexV9egEYM/edit).

@mmvandieren will make the edits and complete a PR

### PR DESCRIPTION
**Pull request by mmvandieren**
### Description

Closes #948

### CODE REVIEW

File: docs/sphinx/examples/cpp/basics/noise_amplitude_damping.cpp [hunk 1]
```diff
@@ -20,12 +20,12 @@ int main() {
   // our damping channel to.
   cudaq::noise_model noise;
 
-  // Amplitude damping channel with `1.0` probability of the qubit
-  // decaying to the ground state.
+  // We define an amplitude damping channel setting to `1.0` the probability of 
+  // the qubit decaying to the ground state.
```
Comment (schweitzpgi) [inline docs/sphinx/examples/cpp/basics/noise_amplitude_damping.cpp:24]:
```suggestion
  // We define an amplitude damping channel setting the probability of 
  // the qubit decaying to the ground state to `1.0`.
```
Reads better, I think.

File: docs/sphinx/examples/cpp/basics/noise_bit_flip.cpp [hunk 2]
```diff
@@ -20,7 +20,8 @@ int main() {
   // these decoherence channels to.
   cudaq::noise_model noise;
 
-  // Bit flip channel with `1.0` probability of the qubit flipping 180 degrees.
+  // We define a bit-flip channel setting to `1.0` probability of the
+  // qubit flipping 180 degrees about the X axis.
```
Comment (schweitzpgi) [inline docs/sphinx/examples/cpp/basics/noise_bit_flip.cpp:24]:
```suggestion
  // We define a bit-flip channel setting the probability of the
  // qubit flipping 180 degrees about the X axis to `1.0`.
```

File: docs/sphinx/examples/cpp/basics/noise_depolarization.cpp [hunk 3]
```diff
@@ -12,18 +12,18 @@
...
 int main() {
 
   // We will begin by defining an empty noise model that we will add
   // our depolarization channel to.
   cudaq::noise_model noise;
 
-  // Depolarization channel with `1.0` probability of the qubit state
-  // being scrambled.
+  // We define a depolarization channel setting to `1.0` the probability of the
+  // qubit state being scrambled.
```
Comment (schweitzpgi) [inline docs/sphinx/examples/cpp/basics/noise_depolarization.cpp:24]: similar, moving 1.0 to the end of the sentence

File: docs/sphinx/using/cpp.rst [hunk 4]
```diff
@@ -20,15 +19,15 @@ Here we see that we can define a custom :code:`struct` that is templated on a :c
...
+operations, like the :code:`x<cudaq::ctrl>(q[0],q[1])` operation which implements a controlled-X gate. We 
 can measure single qubits or entire registers. 
 
 In this example we are interested in sampling the final state produced by this CUDA Quantum kernel. 
 To do so, we leverage the generic :code:`cudaq::sample` function, which returns a data type 
 encoding the qubit measurement strings and the corresponding number of times that string 
 was observed (here the default number of shots is used, :code:`1000`).
 
-To compile and execute this code, we run the following 
+To compile and execute this code, we run the following:
```
Comment (schweitzpgi) [inline docs/sphinx/using/cpp.rst:30]: "The following example illustrates how to compile and execute this code."

Avoids the hanging colon, removes the we.

**Review by schweitzpgi — COMMENTED** [review]
LGTM

File: docs/sphinx/examples/cpp/basics/noise_depolarization.cpp (hunk 3 above)
Comment (bettinaheim) [reply docs/sphinx/examples/cpp/basics/noise_depolarization.cpp:24]: +1

File: docs/sphinx/examples/cpp/basics/noise_phase_flip.cpp [hunk 5]
```diff
@@ -20,11 +20,11 @@ int main() {
   // our phase flip channel to.
   cudaq::noise_model noise;
 
-  // Phase flip channel with `1.0` probability of the qubit
-  // undergoing a phase rotation of 180 degrees (π).
+  // We define a phase-flip channel setting to `1.0` the probability of the
```
Comment (bettinaheim) [inline docs/sphinx/examples/cpp/basics/noise_phase_flip.cpp:23]: nit: Like Eric, I personally find the "... setting the probability of ... to 1.0" much more natural to read.

Comment (mmvandieren) [reply docs/sphinx/examples/cpp/basics/noise_phase_flip.cpp]: the last commit fixes this.

File: docs/sphinx/using/cpp.rst (hunk 4 above)
Comment (mmvandieren) [reply docs/sphinx/using/cpp.rst]: the last commit fixes this.

File: docs/sphinx/examples/cpp/basics/noise_depolarization.cpp (hunk 3 above)
Comment (mmvandieren) [reply docs/sphinx/examples/cpp/basics/noise_depolarization.cpp]: the last commit fixes this.

**Review by bettinaheim — APPROVED** [review]

**Pull request merged**